### PR TITLE
Red team setup instructions

### PR DESCRIPTION
**Pull request by magnologan**
Setup instructions for Red Team (red-ch1.md)

How to install Metasploit container
How to install and use nmap 
How to run nmap scan

### CODE REVIEW

File: trainingportal/static/lessons/redTeam/red_ch1.md [hunk 1]
```diff
@@ -3,17 +3,60 @@ Use nmap to scan for open ports of the provided host, identify applications runn
...
+
+`docker pull securecodebox/nmap`
+
+OR for Debian/Ubuntu:
+
+`apt-get install nmap`
+ 
+OR for RedHat/CentOS:
+
+`rpm -vhU https://nmap.org/dist/nmap-7.80-1.x86_64.rpm`
```
Comment (paul-ion) [inline trainingportal/static/lessons/redTeam/red_ch1.md:21]: On Amazon EC2 I found that `sudo yum install nmap` works as well

File: trainingportal/static/lessons/redTeam/red_ch1.md [hunk 2]
```diff
@@ -3,17 +3,60 @@ Use nmap to scan for open ports of the provided host, identify applications runn
...
 
+###### Pull metasploit docker image from DockerHub
 
-#### Instructions for Completing the Challenge
-- Run nmap
-- Run metasploit or a different exploit script to take advantage of a vulnerability in the first identified app
-- Execute commands remotely
-- Find the flag program in the current directory and use it to sign the challenge salt: `./flag <SALT>`
\ No newline at end of file
+`docker pull metasploitframework/metasploit-framework`
```
Comment (paul-ion) [inline trainingportal/static/lessons/redTeam/red_ch1.md:44]: Should we advise that metasploit is installed on a EC2 that is internet accessible to allow for the meterpreter to run?

File: trainingportal/static/lessons/redTeam/red_ch1.md (hunk 1 above)
Comment (magnologan) [reply trainingportal/static/lessons/redTeam/red_ch1.md]: Updated with yum command.

File: trainingportal/static/lessons/redTeam/red_ch1.md (hunk 2 above)
Comment (magnologan) [reply trainingportal/static/lessons/redTeam/red_ch1.md:46]: Sounds good. Added message before the docker pull.

**Pull request merged**